### PR TITLE
Update kabanero-cli.adoc

### PR DESCRIPTION
**Pull request by t-koman**
For #368 - updates for describe command

### CODE REVIEW

File: ref/general/reference/kabanero-cli.adoc [hunk 1]
```diff
@@ -247,6 +247,38 @@ kabanero cli version: 0.8.0
...
+project:  kabanero
+source:  https://raw.githubusercontent.com/kabanero-io/kabanero-command-line-services/master/index.yaml
+image:  docker.io/kabanerobeta/java-spring-boot2
+status:  active
+digest check:  matched
+kabanero digest:  5343a17b5a2d1923755d5853f4f0507f4ef1cc429238768d9b17c34dee3b829a
+image digest:  5343a17b5a2d1923755d5853f4f0507f4ef1cc429238768d9b17c34dee3b829a
+---
+
+Notice that the describe command allows you to see that the kabanero digest matches the image digest.
```
Comment (jantley-ibm) [inline ref/general/reference/kabanero-cli.adoc:280]: suggest adding backticks around the word describe in this sentence

File: ref/general/reference/kabanero-cli.adoc [hunk 2]
```diff
@@ -247,6 +247,38 @@ kabanero cli version: 0.8.0
...
+
+Example:
+
+---
+kabanero describe java-spring-boot2 0.3.28
+---
+
+Example result:
+---
+user-mbp kabanero-command-line =>./build/kabanero describe java-spring-boot2 0.3.28
```
Comment (jantley-ibm) [inline ref/general/reference/kabanero-cli.adoc:268]: @s1cyan is this example anonymous enough as it is, or are there some changes we could make?

File: ref/general/reference/kabanero-cli.adoc [hunk 3]
```diff
@@ -247,6 +247,38 @@ kabanero cli version: 0.8.0
...
 
+=== Describe
+
+Run the `kabanero describe` command to list the details of an application stack.
+
+---
+kabanero describe stack-name version-number
+---
+
+Running the describe command produces a list of details that allows for identifying mismatches in the specified application stack.
```
Comment (jantley-ibm) [inline ref/general/reference/kabanero-cli.adoc:258]: suggested change:
Running the `describe` command produces a list of details for the specified application stack. This information is particularly useful when a mismatch has been indicated for the digest values.

File: ref/general/reference/kabanero-cli.adoc (hunk 2 above)
Comment (s1cyan) [reply ref/general/reference/kabanero-cli.adoc:268]: I think the past example results in this doc haven't included that line (268) its just shown the output

File: ref/general/reference/kabanero-cli.adoc (hunk 1 above)
Comment (s1cyan) [reply ref/general/reference/kabanero-cli.adoc:280]: Im not sure if the markdown here translates to something else on the kabanero-io page, but when I view this file, the spacing on this is all messed up

File: ref/general/reference/kabanero-cli.adoc (hunk 2 above)
Comment (t-koman) [reply ref/general/reference/kabanero-cli.adoc:268]: @jantley-ibm Good question.  the java-spring-boot2 might need to be more anonymous..

File: ref/general/reference/kabanero-cli.adoc (hunk 1 above)
Comment (t-koman) [reply ref/general/reference/kabanero-cli.adoc:280]: @jantley-ibm Thanks for catching this.

**Comment by t-koman** [comment]
@s1cyan @jantley-ibm Claudia and Jeff, I've fixed the formatting of this topic and made the appropriate editing changes.....Please review

**Comment by t-koman** [comment]
@s1cyan thanks Claudia for the review.  @jantley-ibm Jeff take a look and approve also.. and I can then merge the pull request ....?

**Pull request merged**